docs/cita/install.md

```diff
@@ -270,7 +263,7 @@ $ tail -100f cita_secp256k1_sha3_node0/test-chain/0/logs/cita-jsonrpc.log
   {
     "jsonrpc": "2.0",
     "id": 74,
-    "result": "0x3"
+    "result": "0x1"
```

Comment:
Two nodes has been started in above commands.  The result should be 2.

Reply:
`peerCount`  means how many nodes connected to me, so it would be `1`, not `2`.

Reply:
OK